frontend/src/components/Header.tsx

```diff
@@ -174,10 +174,7 @@ const HeaderNoCtx: React.FC<ContextType> = (props) => {
                 alt="Crossfeed Icon Navigate Home"
               />
             </Link>
-            <div className={classes.mdNav}>{desktopNavItems.slice(0, 3)}</div>
```

Comment:
The original code hid the "scan" navitem when the screen was <1280px. Instead of hiding one of the NavItems, I made the sidebar accessible until all navitems can be shown side by side.